config/tink-server/role.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
This looks like a duplicate of `config/tink-server/rbac/role.yaml` right?

Reply:
yep, removed